hazelcast/src/test/java/com/hazelcast/test/annotation/IntegrationTest.java

```diff
@@ -0,0 +1,10 @@
+package com.hazelcast.test.annotation;
+
+/**
+ * Tests which should be executed after a final JAR is packaged.
+ * These tests are executed by the Fail-Safe Maven plugin.
+ * (other tests are executed by the Surefire plugin)
+ *
+ */
+public final class IntegrationTest {
```

Comment:
What does IDEA do, when you execute all tests from the package? Are those integration tests picked up? If so I would like to keep the annotation, because then I can use it to ignore those tests with a batch script I have, like I do it with the `NightlyTest` category, when executing my local code coverage runs.